crates/router/src/core/payments.rs

```diff
@@ -36,11 +38,17 @@ pub use self::operations::{
     PaymentUpdate,
 };
 use self::{
+    conditional_configs::perform_decision_management,
     flows::{ConstructFlowSpecificData, Feature},
+    helpers::get_key_params_for_surcharge_details,
     operations::{payment_complete_authorize, BoxedOperation, Operation},
     routing::{self as self_routing, SessionFlowRoutingInput},
 };
-use super::errors::StorageErrorExt;
+use super::{
+    errors::StorageErrorExt,
+    payment_methods::surcharge_decision_configs::perform_surcharge_decision_management_for_session_flow,
+    utils::persist_individual_surcharge_details_in_redis,
```

Comment:
Remove here and add self in utils and use utils::fn directly, for payment_method also follow the same method.